### ISSUE TITLE
Couldn't find the target note.

### ISSUE THREAD

**Issue by DumbBoy2** [Ajatt-Tools/mpvacious#17]
**Environment:**
 - OS: Windows 8.1
 - mpv version: latest (redownloaded yesterday)

**Describe the bug**
As mentioned in the title everything works fine until I return to mpv to press "m" and that error pops up. I think it's due to some kind of error in the config file.

**To Reproduce**
Follow the exact same method you described up until you get to the "press m" part.

**Expected behavior**
Shoul have taken and added screenshot and audio recording to the anki note. Instead it just says "Couldn't find the target note".

**Screenshots**
here is my config file and a bunch of screenshots  included my yomichan settings. 

![config](https://user-images.githubusercontent.com/72475912/103368601-4830bb80-4ac8-11eb-8509-a8048f323338.PNG)
![config2](https://user-images.githubusercontent.com/72475912/103368602-4961e880-4ac8-11eb-9d74-5450e9f80327.PNG)
![Config3](https://user-images.githubusercontent.com/72475912/103368605-4961e880-4ac8-11eb-801b-9dc86141a2a0.PNG)
![config4](https://user-images.githubusercontent.com/72475912/103368606-49fa7f00-4ac8-11eb-95e8-036e257cbc76.PNG)
![config5](https://user-images.githubusercontent.com/72475912/103368607-49fa7f00-4ac8-11eb-8a3d-ab2f657d853d.PNG)
![config6](https://user-images.githubusercontent.com/72475912/103368608-4a931580-4ac8-11eb-98c9-0a896805761e.PNG)

**Comment by tatsumoto-ren** [comment]
Thank you for the report. Can you add notes normally (by pressing `n` instead of `m`)? 
If not, [curl](https://curl.se/) might be misconfigured. On windows 8.1 `curl` is not installed by default and must be installed manually.

**Comment by DumbBoy2** [comment]
Thank you for the reply. When i press "n" it says: "Error: Ankiconect isn't running" but it's not true because i can create cards trough yomichan. I tried installing curl but I get the same error. I'm not exactly an expert in programming so I might just have installed it the wrong way. How do I do that exactly? I've downloaded curl, put the folder into programs(x86) e executed the curl.exe file. Something wrong?
